docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -577,6 +577,40 @@ GBD has state-level all-cause mortality, does FBD forecast at the US
 state level yet? Not necessary right now, but good to know for the
 future.
 
+When a simulant who is the reference person in a non-GQ household dies,
+the oldest remaining simulant in their household is assigned to be the reference person.
+All other simulants in the household are assigned a new relationship with these steps:
+
+#. If the new reference person is this simulant's tracked parent (i.e. :code:`parent_ids`),
+   the simulant is assigned 'Biological child.'
+#. Otherwise, the simulant is assigned the value in the :code:`relationship_to_new_reference_person`
+   column in the CSV data file below, from the row where the
+   :code:`relationship_to_old_reference_person` column matches this simulant's current relationship
+   attribute and the :code:`new_reference_person_relationship_to_old_reference_person` column
+   matches the previous relationship attribute of the new reference person.
+#. If there is no such row in the file (which would only happen with very strange combinations,
+   e.g. a person having two spouses), the simulant is assigned 'Other nonrelative.'
+
+:download:`reference_person_update_relationship_mapping.csv`
+
+This file is not exact:
+
+* There is not always sufficient information to uniquely determine a new relationship. We err
```

Comment:
when you say not enough to "uniquely determine", does that mean that sometimes the csv would suggest 2+ different new relationships, or sometimes that it would not suggest a new relationship?

(the latter wouldnt be a problem if paired with line 592 / your 'other nonrelative' rule, so i think that's probably what you mean?)

Reply:
Apologies for the confusion -- I did not mean either of these things.

The CSV will never suggest more than 1 relationship. However, it will not always suggest a relationship. This is the reason for the 'other nonrelative' rule.

However, the "uniquely determine" language and indeed this entire list of bullets were intended to be assumptions/limitations regarding the _construction_ of the file, not its use. (Only important from a research perspective, not an engineering one.) I will change the text above the list to make this more clear.